source/blog/2019-05-21-monetizing-open-source.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Monetizing Open Source"
+authorName: Jonathan H. Wage
+authorEmail: jonwage@gmail.com
+permalink: /2019/05/21/monetizing-open-source.html
+---
+
+In our quest to make Doctrine financially sustainable, we have created
+Doctrine Company, LLC for the team to do business under. We've had a strong
```

Comment:
"for the team to do business under" can be rewritten as: "under which the team can do business" to avoid ending a sentence with a preposition. See: https://english.stackexchange.com/questions/115958/how-else-should-i-avoid-using-under-at-the-end-of-the-sentence

I might also suggest "under which the team can conduct business" if you think that reads well.